src/parser/monk/mistweaver/modules/spells/InvokeYulon.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+
+import Analyzer, { SELECTED_PLAYER, SELECTED_PLAYER_PET } from 'parser/core/Analyzer';
+import Events from 'parser/core/Events';
+import Combatants from 'parser/shared/modules/Combatants';
+
+import SpellIcon from 'common/SpellIcon';
+import StatisticBox, { STATISTIC_ORDER } from 'interface/others/StatisticBox';
+
+import { formatNumber } from 'common/format';
+import { TooltipElement } from 'common/Tooltip';
+
+const debug = false;
+
+class InvokeYulon extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  petID = null;
+  soothHealing = 0;
+  envelopHealing = 0;
+  healing = 0;
+
+  constructor(...args) {
+    super(...args);
+    this.addEventListener(Events.heal.by(SELECTED_PLAYER).spell(SPELLS.ENVELOPING_BREATH), this.handleEnvelopingBreath);
+    this.addEventListener(Events.heal.by(SELECTED_PLAYER_PET).spell(SPELLS.SOOTHING_BREATH), this.handleSoothingBreath);
+    this.addEventListener(Events.summon.by(SELECTED_PLAYER), this.handleSummon);
```

Comment:
You can add .spell here as well, with the summon event ability id for yulon. This will fix the error I mentioned in the last review about incidental summons (jade totem, trinkets, random bs) breaking the module